R/MeasureFairness.R

```diff
@@ -0,0 +1,54 @@
+#' @title Fairness Measure
+#' @name mlr_measures_fairness
+#'
+#' @description
+#' This measure specializes [Measure] for Fairness Measure problems:
+#' Users could use [groupwise_abs_diff()], [groupwise_diff()], or [groupwise_quotient()] to evaluate
+#' the fairness measures based on base_measures. For example, the false positive rate bias or
+#' the equalized odds ratios. More examples could viewed on mlr3fairness wikipage.
+#'
+#' Predefined measures can be found in the [dictionary][mlr3misc::Dictionary] [mlr_measures].
+#' Predefined operations can be found in the [mlr3fairness].
```

Comment:
some incomplete sentences here.

Reply:
Not sure what "incomplete" means. Do you want me to add some clarifications on measures and operations?

Reply:
`for Fairness Measure problems` -> t`to allow for measuring fairness`.

Reply:
Got it, corrected my grammar.